src/reset-password/ResetPasswordPage.jsx

```diff
@@ -53,7 +53,11 @@ const ResetPasswordPage = (props) => {
   const validatePasswordFromBackend = async (password) => {
     let errorMessage = '';
     try {
-      errorMessage = await validatePassword(password);
+      const payload = {
+        reset_password_page: true,
```

Comment:
Do we still need this since we have a separate feature flag at the backend?

Reply:
yes, this is required along with the feature flag to enable validation only for reset password page

Reply:
We are using the same Validation API endpoint for registration and sign-in validations as well